### PR TITLE
Remove `OrgActivity`

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/orgs/views.py [hunk 1]
```diff
@@ -1226,14 +1226,14 @@ def derive_url_pattern(cls, path, action):
             return r"^%s/%s/((?P<submenu>[A-z]+)/)?$" % (path, action)
 
         def has_permission(self, request, *args, **kwargs):
+            self.org = self.request.org
```
Comment (rowanseymour) [inline temba/orgs/views.py:1229]: `OrgPermsMixin` usually sets `self.org` so keeping that straight

File: temba/orgs/views.py [hunk 2]
```diff
@@ -1333,20 +1333,17 @@ def derive_menu(self):
 
                     menu.append(self.create_menu_item(name=_("Archives"), items=items, inline=True))
 
-                child_orgs = Org.objects.filter(parent=self.org, is_active=True, plan=settings.PARENT_PLAN).order_by(
-                    "name"
-                )
-
+                child_orgs = self.org.children.filter(is_active=True).order_by("name")
```
Comment (rowanseymour) [inline temba/orgs/views.py:1336]: can remove filtering on plan="parent" because soon that will be redundant

File: temba/orgs/views.py [hunk 3]
```diff
@@ -3402,7 +3399,7 @@ def derive_formax_sections(self, formax, context):
             user = self.request.user
             org = self.request.org
 
-            if not (org.parent and org.plan == settings.PARENT_PLAN):
+            if not (org.parent and org.plan == "parent"):
```
Comment (rowanseymour) [inline temba/orgs/views.py:3402]: not sure what to do here.. it's the last place hanging on to our specific plan logic

Comment (norkans7) [reply temba/orgs/views.py:3402]: `Org.show_plan` method that can be overridden??

**Pull request merged**